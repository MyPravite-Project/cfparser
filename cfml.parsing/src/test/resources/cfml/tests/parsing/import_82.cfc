import(Event, local.uploadPathBild&file.serverfile);