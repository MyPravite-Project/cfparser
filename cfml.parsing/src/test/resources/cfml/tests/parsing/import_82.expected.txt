/*===TOKENS===*/
IMPORT                   <import>
'('                      <(>
IDENTIFIER               <Event>
','                      <,>
IDENTIFIER               <local>
'.'                      <.>
IDENTIFIER               <uploadPathBild>
'&'                      <&>
FILE                     <file>
'.'                      <.>
IDENTIFIER               <serverfile>
')'                      <)>
';'                      <;>
/*===TREE===*/
(scriptBlock
  (element
    (statement
      (startExpression
        (baseExpression
          (unaryExpression
            (memberExpression
              (functionCall
                (identifier import)
                (
                (argumentList
                  (argument
                    (startExpression
                      (baseExpression (unaryExpression (memberExpression (identifier Event))))
                    )
                  )
                  ,
                  (argument
                    (startExpression
                      (baseExpression
                        (baseExpression
                          (unaryExpression
                            (memberExpression (identifier local) . (identifier uploadPathBild))
                          )
                        )
                        &
                        (baseExpression
                          (unaryExpression
                            (memberExpression (identifier (cfmlFunction file)) . (identifier serverfile))
                          )
                        )
                      )
                    )
                  )
                )
                )
              )
            )
          )
        )
      )
      ;
    )
  )
)
/*======*/
/*===DECOMPILE===*/
import(Event, local.uploadPathBild & file.serverfile)
/*======*/